src/main/java/com/codeofus/rent_a_park/services/PersonService.java

```diff
@@ -27,12 +30,19 @@ public Person addNewPerson(Person person) {
         return personRepository.save(person);
     }
 
+    @Cacheable(value = "persons")
+    public List<Person> getAllPersons() {
+        return personRepository.findAll();
+    }
+
+    @Cacheable(value = "persons", key = "#pageable")
```

Comment:
What if one person is edited, will this update the person

Reply:
If the person is edited, for the same pageable, the method `getAll` will return the cached answer and it will include the "old" version of a person (if exists in the cache). If it is called with the pageable that is not cached, it will update the new version. So, the whole list is cached.

Reply:
fixed eviction of cache for list of persons if one updated